.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Handle stale issues and PRs
+on:
+  schedule:
+    - cron: "0 11 * * *"
+  push:
+    branches: [stale-bot]
+jobs:
+  handle-stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 90
+          days-before-issue-close: -1
```

Comment:
Just thinking if we should add a link to the stale Action somewhere in this yml for users to quickly check the guide for it or write a comment here that -1 means they are not closed automatically. 

Also, okay if we don't take any action, just sharing what I had a thought of.

Reply:
Thanks, @pankajkoti , addressed in 03eb12c75f666f7e625b08d81ad9b0aee708a9b2